plugins/package-managers/spm/src/main/kotlin/SpmModel.kt

```diff
@@ -43,16 +44,16 @@ data class SwiftPackageDependencies(
     val url: String,
     val version: String,
     val path: String,
-    val dependencies: List<LibraryDependency>
-)
-
-@Serializable
-data class LibraryDependency(
```

Comment:
Commit message: This is not turning `AppDependency` into a nested class, but `LibraryDependency`.

Reply:
In the commit message body "Get rid of the misleading term `App`" still mentions "App".

Reply:
Semi-related: I'm not so happy with the long / nested `org.ossreviewtoolkit.plugins.packagemanagers.spm.SwiftPackageDependencies.Dependency` imports. How about also renaming `SwiftPackageDependencies` to `SwiftPackage` (as it contains more than just the `dependencies` anyway) and then using `SwiftPackage.Dependency` instead of `Dependency`?

Reply:
I've amended the commit at the bottom to rename to `SwiftPackage` instead of to `SwiftPackageDependencies`.